CHANGELOG.md

```diff
@@ -17,6 +17,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   code
 - README has been updated with example SSLContext setup and it's use in Conjur
   class constructors
+- Maven pom.xml file includes configuration profiles for deploying to Sonatype OSSRH 
+(OSS Repository Hosting)  ([cyberark/conjur-api-java#6](https://github.com/cyberark/conjur-api-java/issues/6))
+- Introduced [upgrade instructions](https://github.com/cyberark/conjur-api-java/UPGRADING.md) to provide instructions for
+  upgrading to 3.0.0, or make use of published artifacts. These can be found in
+ `UPGRADING.md`. ([cyberark/conjur-api-java#77](https://github.com/cyberark/conjur-api-java/issues/77))
+
+### Changed 
+- Revised package references from `net.conjur.api` to `com.cyberark.conjur.api`
+- Revised visioning in `pom.xml` to use `SNAPSHOT` as the default
```

Comment:
visioning => versioning?

by "use SNAPSHOT as the default", do you mean that we publish with every commit on the master branch? typically we prefer to only publish on tags - can you please clarify this point?

Reply:
We publish snapshot builds on every build of master. Additionally, as long as `SNAPSHOT` is in the version, it will not allow us to accidentally publish a release. The release script is set to only run on a tagged release, and it removes the `SNAPSHOT` suffix automatically.

Reply:
This change shouldn't affect users in the slightest, so I'll strike it.